util/misc/chpl2pdf

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
```

Comment:
Please add a description of what this script does and how to use it. OK if brief.